Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -781,11 +781,7 @@ extension NavigationViewController: NavigationServiceDelegate {
         let preventRerouting = navigationService.delegate?.navigationService(navigationService, shouldPreventReroutesWhenArrivingAt: destination) ?? RouteController.DefaultBehavior.shouldPreventReroutesWhenArrivingAtWaypoint
         let userArrivedAtWaypoint = progress.currentLegProgress.userHasArrivedAtWaypoint && (progress.currentLegProgress.distanceRemaining <= 0)
 
-        if snapsUserLocationAnnotationToRoute && (!userArrivedAtWaypoint || preventRerouting) {
-            ornamentsController?.labelCurrentRoad(at: rawLocation, suggestedName: roadName(at: rawLocation), for: location)
-        } else  {
-            ornamentsController?.labelCurrentRoad(at: rawLocation, suggestedName: roadName(at: rawLocation))
-        }
+        ornamentsController?.labelCurrentRoadName(suggestedName: roadName(at: rawLocation))
```

Comment:
I wonder if we should keep using `rawLocation` in this case. Should we also check whether snapped location is also available and make use of it in first place?

Reply:
Thanks for the suggestion, the snapped location would be checked first.